src/params/progression.jl

```diff
@@ -39,7 +39,9 @@ const severity_dists = [
 
 const death_probs = [0.0, 0.0, 0.0, 0.49]
 
-function sample_severity(rng::AbstractRNG, age::Real)
+const vaccination_effectiveness = [0.0, 0.75, 0.875, 0.92]
```

Comment:
Does these number already include prior immunity to go get infected?

Reply:
numbers are copied from branch of  @eMaerthin

Reply:
yes, but there is already some effectiveness in the infectivity table an they probably account for at least a part of it. In other words: you will likely not get infected because of them, so this should be conditional on the event that you did get infected despite the immunity

Reply:
yes, but these parameters work in case of  subject is vaccinated and he is infected, so subject will be ill less severitly

Reply:
OK, I though this was handling the whole immunity in Marcin's code. Is that OK @eMaerthin ?

Reply:
in Marcin's code there is another part responsible for vaccination with const distribution, but i ignored it.